### DISCUSSION TITLE
Software developer

### DISCUSSION THREAD

**Discussion by amirhemmateenejad** [WeAreMahsaAmini/Core#332] [Introductions]
hi its amir
im a junior software developer 
i know php,node js for backend
dart and flutter for mobile application
and i have exprience with sql databases and mongodb for no sql databases
you can contact with me in telegram 
https://t.me/ahp_35

**Comment by ehsaaanes** [comment]
Salam, lotfan proje #163 ro check konin, age tamayol be hamkari darid baram benevisin, mamnoon
